style.css

```diff
@@ -31,7 +22,44 @@
   background-image: url("./images/background.svg");
   background-size: 100vw;
   background-repeat: no-repeat;
-  overflow: hidden;
+}
+
+.menu-item-style {
+  display: flex;
+  background: #6070ff;
+  mix-blend-mode: multiply;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  color: #fff;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  height: 120vh;
+}
```

Comment:
- [ ] The Menu is more transparent. I would recommend removing `mix-blend-mode: multiply;` and replacing it with `opacity: 0.95` to improve the UI design and also for a better user experience.

Reply:
The changes applied!